File: LV1/1.3/pet/pet.go
package main

import "fmt"

func main() {
	var maxVal, idx int

	for i := 1; i <= 5; i++ {
		var sum, tmp int
		for j := 0; j < 4; j++ {
			fmt.Scanf("%d", &tmp)
			sum += tmp
		}
		if sum > maxVal {
			maxVal = sum
			idx = i
		}
	}
	fmt.Println(idx, maxVal)
}

File: LV1/1.3/pet/pet.cpp
#include <cstdio>

int main() {
    int max_val = 0, idx = 0;

    for (int i = 1; i <= 5; i++) {
        int sum = 0, tmp;
        for (int j = 0; j < 4; j++) {
            scanf("%d", &tmp);
            sum += tmp;
        }
        if (sum > max_val) {
            max_val = sum;
            idx = i;
        }
    }
    printf("%d %d\n", idx, max_val);
}